packages/app-builder-lib/src/asar/unpackDetector.ts

```diff
@@ -82,7 +82,11 @@ export async function detectUnpackedDirs(fileSet: ResolvedFileSet, autoUnpackDir
     const moduleName = path.basename(packageDir)
     if (moduleName === "ffprobe-static" || moduleName === "ffmpeg-static" || isLibOrExe(file)) {
       shouldUnpack = true
-    } else if (!file.includes(".", nextSlashIndex)) {
+    }
+
+    const fileBaseName = path.basename(file)
+    const hasExtension = path.extname(fileBaseName)
+    if (!hasExtension) {
```

Comment:
I think this needs to remain an `else-if` otherwise `isBinaryFileSync` could override `shouldUnpack` for the file-specific logic of
```
moduleName === "ffprobe-static" || moduleName === "ffmpeg-static" || isLibOrExe(file)
```

Reply:
Correct.